sourmash/cli/index.py

```diff
@@ -64,7 +59,7 @@ def subparser(subparsers):
     )
     subparser.add_argument(
         '-f', '--force', action='store_true',
-        help='try loading all files with --traverse-directory'
+        help='try loading all files in provided subdirectories"'
```

Comment:
Assuming default is similar to the rest of the commands + your help text, I'm thinking this means:  If you provide directory, will it load all sigs in that directory. Using `--force` looks in subdirectories within the provided dir? 

Is that the case, or do I have it wrong?

Reply:
whups, nope, I'll fix :)

Reply:
>New behavior: if a directory is provided on the command line, it is automatically traversed for signatures (*.sig). -f still forces loading filenames that do not end with .sig

Ah, nm, I see this in the `conversation` tab. Maybe we can alter help text to better describe?

Reply:
fixed in f0913fb - now "try loading *all* files in provided subdirectories, not just .sig files"